pydy/codegen/ode_function_generators.py

```diff
@@ -317,10 +318,17 @@ def linear_sys_solver(self, v):
 
         if isinstance(v, type(lambda x: x)):
             self._solve_linear_system = v
+            self._linear_sys_solver = v
         elif v == 'numpy':
             self._solve_linear_system = numpy.linalg.solve
+            self._linear_sys_solver = v
         elif v == 'scipy':
             self._solve_linear_system = scipy.linalg.solve
+            self._linear_sys_solver = v
+        elif v == 'sympy':
+            # dummy function
+            self._solve_linear_system = lambda A, b: b
+            self._linear_sys_solver = v
```

Comment:
These `self._linear_sys_solver = v` are missing and are actually a bug fix. Needs to be ported to master even if this isn't merged.